languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -61,4 +70,28 @@ object ImportResolver {
     }
   }
 
+  /**
+    * Given a string representing an http(s) url, retrieve the contents
+    */
+  def httpResolver: ImportResolver =
+    httpResolverWithHeaders(Map.empty[String,String])
+
+  /**
+    * Given a string representing an http(s) url, retrieve the contents
+    * using the supplied headers (which can be an empty map)
+    */
+  def httpResolverWithHeaders(headers: Map[String,String]): ImportResolver = CheckedAtoB.fromCheck { str: String =>
+
+    implicit val sttpBackend = AsyncHttpClientCatsBackend[IO]()
+
+    val responseIO : IO[Response[String]] =
+      sttp.get(uri"$str").headers(headers).send()
+
+    // temporary situation to get functionality working before
+    // starting in on async-ifying the entire WdlNamespace flow
+    val result: Checked[String] = Await.result(responseIO.unsafeToFuture, 15.seconds).body.leftMap(NonEmptyList(_, List.empty))
```

Comment:
ToL possible to return IO[ImportResolver] and leave the dirty work to the caller? I know we're doing this all over the place but worth an ask.